src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -52,14 +52,21 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="1.0.2" />
     <PackageReference Include="Microsoft.AspNetCore.Routing" Version="1.0.5" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.WebApiCompatShim" Version="1.0.5" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.2.1" />
```

Comment:
Might as well upgrade these to the versions of the Telemetry package you are using.

In fact, I would argue that we shouldn't even package a NET full framework compatible version of this package, as it has no benefits for Functions V1 customers. We would want to leave in all of our conditional compilation work so that it is easy to merge into the main branch once that is ready, but no reason to publish a V1 compatible version while they are still separate packages.

Reply:
We talked about it, It is because of the issue that shibayan point out.